Extras/Equilibrium/Launcher

```diff
@@ -12,6 +12,7 @@
 # $4 Boundary Percentage, Take Profit, Deviation
 # $5 Buy Lots
 # $6 Sell Lots
+# $7 Paper trading indicator
```

Comment:
Because this seems to be the last argument in-use, I was thinking to remove `$8 $9` from line 34 below, but I'll let that be your call, @rapmd73, in case you have future plans for them.

Reply:
It really could be and probably should be. I left it in there temporarily as nothing more than a placeholder for testing purposes if I needed to change the format.